var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -367,7 +368,7 @@ class Gdal(CMakePackage, AutotoolsPackage, PythonExtension):
     # depends_on('tiledb', when='+tiledb')
     depends_on("libwebp", when="+webp")
     depends_on("xerces-c@3.1:", when="+xercesc")
-    depends_on("zstd", when="+zstd")
+    depends_on("zstd", when="+zstd")  # TODO only true if internal libtiff is used?
```

Comment:
@rouault does GDAL use ZSTD internally for anything or is it only used to build libtiff?

Reply:
Ah, looks like it's used by the Zarr driver: https://gdal.org/development/building_from_source.html#zstd